src/main/java/com/theelixrlabs/healthcare/controller/DoctorController.java

```diff
@@ -48,21 +43,27 @@ public ResponseEntity<SuccessResponse<DoctorDto>> createDoctor(@Valid @RequestBo
     /**
      * Retrieves a doctor by their UUID.
      *
-     * @param id The ID of the doctor as a String, which will be converted to a UUID.
+     * @param doctorId    The ID of the doctor as a String, which will be converted to a UUID.
      * @return ResponseEntity containing a success response with the retrieved DoctorDto.
      */
-    @GetMapping(DoctorConstants.GET_DOCTOR_BY_ID_ENDPOINT)
-    public ResponseEntity<?> getDoctorById(@PathVariable(DoctorConstants.PATH_VARIABLE_DOCTOR_ID) String id) {
-        UUID doctorId;
-        try {
-            doctorId = UUID.fromString(id);
-        } catch (IllegalArgumentException illegalArgumentException) {
-            throw new CustomException(messageUtil.getMessage(MessageConstants.INVALID_UUID));
-        }
+    @GetMapping(DoctorConstants.DOCTOR_BY_ID_ENDPOINT)
+    public ResponseEntity<?> getDoctorById(@PathVariable(DoctorConstants.PATH_VARIABLE_DOCTOR_ID) String doctorId) throws CustomException {
         DoctorDto doctorDto = doctorService.getDoctorById(doctorId);
         return new ResponseEntity<>(new SuccessResponse<>(true, doctorDto), HttpStatus.OK);
     }
 
+    /**
+     * Deletes a doctor by their ID.
+     *
+     * @param doctorId    The ID of the doctor to delete, obtained from the URL path.
+     * @return ResponseEntity containing a SuccessResponse indicating the success status of the deletion operation.
+     * @throws CustomException    If there's an issue with deleting the doctor.
+     */
+    @DeleteMapping(DoctorConstants.DOCTOR_BY_ID_ENDPOINT)
+    public ResponseEntity<SuccessResponse<String>> deleteDoctorById(@PathVariable String doctorId) throws CustomException {
+        return new ResponseEntity<>(new SuccessResponse<>(true, doctorService.deleteDoctorById(doctorId)), HttpStatus.OK);
```

Comment:
does your success response looks like below
{
success : true,
responseBody : " Doctor with id : {0} deleted Successfully."
}

If yes then do you think responseBody is a correct term for a single string message ?

Reply:
Resolved.